apps/crunchy-postgres/liveness/run_litmus_test.yml

```diff
@@ -4,19 +4,20 @@ kind: Job
 metadata:
   generateName: liveness-pg-
   namespace: litmus
-
 spec:
   template:
     metadata:
       name: liveness
-   
+      labels:
+        liveness: node-chaos
     spec:
       restartPolicy: Never
+      nodeSelector:
+        kubernetes.io/hostname: gke-e2e-cstor-dev-1-default-pool-224a4771-bfl4
```

Comment:
Can we use placeholder here and replace.

Reply:
ok